frontend/src/components/LogIn/LogIn.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import LogIn , { LogInProps } from './LogIn';
```

Comment:
If you want to configure storybook, add folowing imports:
```
import { Provider } from 'react-redux';
import { store } from '../../app/store';
```

Reply:
thanks! 💯